src/Stryker.Solutions/Stryker.Solutions.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
nit

I'm not sure why a new project is necessary just for the solution file handling, but we can refactor that later if we decide we don't need it